.github/workflows/tox.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Upload build
         uses: actions/upload-artifact@v2
         with:
-          name: build-${{ matrix.python-version }}
+          name: build-${{ matrix.python-version }}-${{ github.sha }}
```

Comment:
Given the fact that every change is on a topic branch (since we've protected `main`), and also few people remember the SHAs of head commits of topic branches, maybe include `{{ github.ref }}` instead, or both (as in `build-${{ matrix.python-version }}-${{ github.ref }}-${{ github.sha }}`)

Reply:
ref is containing forward slashes, updating to ref_name, which is more convenient.

Reply:
oh well, ref_name also contains it. Gonna remove ref. It's fine enough.

Reply:
Not quite sure what you mean; I just tried this and `github.ref_name` does appear to contain just the branch name:

```
name: "build-${{ matrix.python-version }}-${{ github.ref_name }}-${{ github.sha }}"
```

Reply:
it does contain `something/branchname`. So that kinda is annoying, as `/` is recognized as directory separator. I don't want to deal with substitutions.

Reply:
Er, no it doesn't? See https://github.com/fghaas/docs/actions/runs/1875290546

Reply:
https://github.com/citynetwork/docs/actions/runs/1865627547

Reply:
Correct.